test/cockroachdb_test.go

```diff
@@ -0,0 +1,82 @@
+package helm_testing
+
+import (
+	"path/filepath"
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/helm"
+	"github.com/gruntwork-io/terratest/modules/k8s"
+	"github.com/stretchr/testify/require"
+	rbacv1 "k8s.io/api/rbac/v1"
+)
+
+func Test_Role(t *testing.T) {
+	helmChartPath, err := filepath.Abs("../cockroachdb")
+	require.NoError(t, err)
+	require.NotNil(t,helmChartPath)
+
+	t.Run("Test TLS Enabled but certs not provided", func(t *testing.T) {
+		options := &helm.Options{
+			SetValues: map[string]string{
+				"tls.enabled": "True",
+				"tls.certs.provided": "False",
+				"tls.certs.certManager": "False",
+			},
+			KubectlOptions: k8s.NewKubectlOptions("", "", "test-deployment"),
```

Comment:
Since this test is only rendering templates, not running them, I don't think `KubectlOptions` is needed, even though the terratest example includes it. The tests seem to pass without it.